packages/hash/api/src/model/knowledge/org.model.ts

```diff
@@ -0,0 +1,268 @@
+import { GraphApi } from "../../graph";
+import {
+  OrgModel,
+  EntityModel,
+  EntityModelCreateParams,
+  AccountFields,
+  EntityTypeModel,
+} from "..";
+import {
+  generateSchemaBaseUri,
+  generateWorkspaceEntityTypeSchema,
+  generateWorkspacePropertyTypeSchema,
+  workspaceAccountId,
+  workspaceTypesNamespaceUri,
+} from "../util";
+
+// Generate the schema for the organization name property type
+export const orgNamePropertyType = generateWorkspacePropertyTypeSchema({
+  title: "Organization Name",
+  possibleValues: [{ primitiveDataType: "Text" }],
+});
+
+export const orgNamedBaseUri = generateSchemaBaseUri({
+  namespaceUri: workspaceTypesNamespaceUri,
+  kind: "propertyType",
+  title: orgNamePropertyType.title,
+});
+
+// Generate the schema for the org size property type
+export const orgSizePropertyType = generateWorkspacePropertyTypeSchema({
+  title: "Organization Size",
```

Comment:
Is this like "big", "small"? Or should this be a `number` not `text`?

Reply:
It's this pre-defined enum:
https://github.com/hashintel/hash/blob/2243b6fbdd96f5b3cebfcc3b09bf23d5144af5ce/packages/hash/api/src/model/knowledge/org.model.ts#L94-L99
Perhaps we should add a todo to constrain the data type here as well

Reply:
Good question, that would be better and more reliable for analytics down the line. I don't know of the historic reason for this range of sizes, though

Reply:
added in 82ec61571aaa69d68269ba2b11be9d11c3c65656